CircleSpace/CircleSpace/Views/Shared/_PageBox.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model CircleSpaceGeneralModels.Models.PageModel
+
+
+<div class="pageBox">
+
+
+
+    <table style="width:100%">
+        <tr>
+            <th>Contributors</th>
+            <th>Emails</th>
+            <th>Route</th>
+        </tr>
+        <tr>
+            <td>
+                @foreach(var contributors in Model.Contributors)
+                {
+                    @contributors.Username;
+                }
+
+            </td>
+            <td>
+                @foreach(var emails in Model.Contributors)
+                {
+                    @emails.Email;
```

Comment:
You are putting multiple Contributers into a single row of a table.
Don't worry about contributor's. Just put the owner of the page in the table.